faq.md

```diff
@@ -5,4 +5,30 @@ css_id: faq
 
 # Frequently asked questions
 
+### **What makes Uptane different from other SOTA security mechanisms?**
 
+
+
+
+
+### **How does Uptane work with other systems and protocols?**
+
+
```

Comment:
Other mechanisms for performing update such as XXX, YYY, ZZZ, are compatible with Uptane.  Uptane can use
any mechanism for data transport and is designed to provide strong security guarantees even if the 
underlying network or transport mechanism is compromised.  If an automaker wants to move to a secure
update system, keeping their existing update system as a transport for Uptane is an effective way to do so.